client/web-sveltekit/src/lib/repo/FileHeader.svelte

```diff
@@ -63,7 +63,7 @@
 </script>
 
 <div class="header" use:sizeToFit={{ grow, shrink }}>
-    <h2 on:copy={stripSpaces}>
+    <h2 on:copy={stripSpaces} data-testid="header-path">
```

Comment:
I suggest to be even more specific hear and include the component name in the test id. Something like

```suggestion
    <h2 on:copy={stripSpaces} data-testid="file-header-path">
```